lib/modules/settingsNavigation.js

```diff
@@ -33,11 +32,59 @@ module.options = {
 	},
 };
 
+const getOptionsURL = (hash = '') => new URL(hash, getURL('options.html'));
+const optionsPage = location.origin === getOptionsURL().origin;
+
+function checkStandaloneRedirect() {
+	// Testing requires access to the options page,
+	// and I couldn't figure out a way to get find the extension URL in nightwatch
+	if (location.hash.startsWith('#res:settings-redirect-standalone-options-page')) {
+		window.stop();
+		location.href = getOptionsURL(location.hash.replace('-redirect-standalone-options-page', '')).href;
+	}
+}
```

Comment:
May be a bit slower and more unreliable than using checking webRequests, but redirecting with it would require 
 the `webRequestBlocker` permission.